src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/Migrations/20251001163656_EES6080_RenameExpiryToExpiresInPreviewTokensTable.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Migrations
+{
+    /// <inheritdoc />
+    public partial class EES6080_RenameExpiryToExpiresInPreviewTokensTable : Migration
```

Comment:
Minor one but Lee recently suppressed the warnings in all migrations with

`// ReSharper disable once InconsistentNaming`

so you could add in the same above the class declaration here to suppress the warning and be consistent with our other migrations.

We should look at changing our naming format to be consistent! - I've added it to the list of things to discuss. 🙂